java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireJavaParser.java

```diff
@@ -49,7 +50,7 @@
  */
 public class SurefireJavaParser implements BatchExtension {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(SurefireJavaParser.class);
+  private static final Logger LOGGER = Loggers.get(SurefireJavaParser.class);
   private final ResourcePerspectives perspectives;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "ResourcePerspectives"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)